host/workflowidratelimit_test.go

```diff
@@ -23,14 +23,86 @@
 package host
 
 import (
+	"flag"
+	"testing"
+
 	"github.com/pborman/uuid"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"github.com/uber/cadence/common/clock"
+	"github.com/uber/cadence/common/dynamicconfig"
+	"github.com/uber/cadence/common/persistence"
+	pt "github.com/uber/cadence/common/persistence/persistence-tests"
 
 	"github.com/uber/cadence/common"
 	"github.com/uber/cadence/common/types"
 )
 
-func (s *IntegrationSuite) TestWorkflowIDSpecificRateLimits() {
+func TestWorkflowIDRateLimitIntegrationSuite(t *testing.T) {
+	flag.Parse()
+
+	clusterConfig, err := GetTestClusterConfig("integration_wfidratelimit_cluster.yaml")
+	if err != nil {
```

Comment:
consider using `require.NoError` instead

Reply:
Changed to require.NoError